libraries/commerce/product/subscriptions/index.js

```diff
@@ -1,6 +1,8 @@
 import { hex2bin } from '@shopgate/pwa-common/helpers/data';
 import showModal from '@shopgate/pwa-common/actions/modal/showModal';
-import { historyPop } from '@shopgate/pwa-common/actions/router';
+import { historyPop, historyPush } from '@shopgate/pwa-common/actions/router';
```

Comment:
Can you please change these imports to use the `@shopgate/engage` package, please?